xbmc/pvr/channels/PVRChannelGroups.cpp

```diff
@@ -71,6 +71,9 @@ bool CPVRChannelGroups::Update(const CPVRChannelGroup &group, bool bUpdateFromCl
     if (group.GroupID() > 0)
       updateGroup = GetById(group.GroupID());
 
+    if (!updateGroup && group.IsInternalGroup() && (updateGroup = GetGroupAll()) && bUpdateFromClient)
```

Comment:
please use the following here, because we want to update the internal group, like we do for all other groups, with the data we've fetched from the database.

```c++
if (group.IsInternalGroup())
  updateGroup = GetGroupAll();
else if (group.GroupID() > 0)
  updateGroup = GetById(group.GroupID());

...
```